static/js/admin.js

```diff
@@ -94,7 +94,7 @@ function get_non_default_streams_names(streams_data) {
 
     _.each(streams_data.streams, function (stream) {
         if (default_streams_names.indexOf(stream.name) < 0) {
-           non_default_streams_names.push(stream.name);
+            non_default_streams_names.push(stream.name);
```

Comment:
I'd put these whitespace reformatting things in their own commit (maybe along with the similar stuff below)